pandasai/smart_datalake/__init__.py

```diff
@@ -280,10 +281,12 @@ def chat(self, query: str):
                 self.logger.log("Using cached response")
                 code = self._cache.get(self._get_cache_key())
             else:
+                prompt_cls = GeneratePythonCodePrompt
                 default_values = {
                     # TODO: find a better way to determine the engine,
                     "engine": self._dfs[0].engine,
                     "save_charts_path": self._config.save_charts_path.rstrip("/"),
+                    "output_type_hint": prompt_cls.get_output_type_hint(output_type),
```

Comment:
I suggest, instead of instantiating a static method here to pass `output_type` and then we assign the right path in the `__init__` method of the `GeneratePythonCodePrompt` class, like this:

```
def __init__(self, **kwargs):
        default_import = "import pandas as pd"
        engine_df_name = "pd.DataFrame"

        self.set_var("default_import", default_import)
        self.set_var("engine_df_name", engine_df_name)

        # assign it here
        self.set_var("output_type_hint", self.get_output_type_hint(kwargs["output_type"]))
```

Reply:
@gventuri 
Done. 
I thought to make exactly the same initially, but i was confusing a bit about `.set_var()` is called both from [`SmartDatalake._get_promt()`](https://github.com/gventuri/pandas-ai/blob/0eca724461843a272eeac929df042d4db52160be/pandasai/smart_datalake/__init__.py#L229) and [`GeneratePythonCodePrompt`'s `__init__()`](https://github.com/gventuri/pandas-ai/blob/0eca724461843a272eeac929df042d4db52160be/pandasai/prompts/generate_python_code.py#L78). And also that `__init__()` hadn't accepted keywords parameters (**kwargs) as well. So i've hesitated to add passing of `output_type`.

Fixed.